torch/_export/serde/serialize.py

```diff
@@ -943,12 +944,23 @@ def serialize(self, graph_module: torch.fx.GraphModule) -> GraphModule:
 
 
 class ExportedProgramSerializer:
-    def __init__(self, opset_version: Optional[Dict[str, int]] = None):
+    def __init__(
+        self,
+        opset_version: Optional[Dict[str, int]] = None,
+        convert_to_cpu: bool = False,
```

Comment:
Do we need to store the original devices for each tensor somewhere so that we could retrieve the original devices?